.github/workflows/update-sources.main.yaml

```diff
@@ -67,6 +67,7 @@ jobs:
         git rev-parse HEAD > ../head
         popd
         rm -rf upstream/.git
+        git add upstream head .konflux
         git add upstream head
```

Comment:
you can just do 
git add upstream head .konflux and remove another git add

Reply:
Done